app/views/content/edit/_change_notes.html.erb

```diff
@@ -3,9 +3,9 @@
       label: {
         text: t("content.edit.change_note.title"),
       },
-      id: "revision-change-note",
+      id: "edition-change-note",
```

Comment:
I guess this was renamed due to us using edition instead? I believe it got it it's original id from it having a name of `revision[change_note]`. I think it might be more consistent with other fields if had an id of `change-note-field` since we don't reference edition in the name (and I assume we're reserving the actual "change-note" id for linking from index).

Reply:
Sure, I can do another PR to change this.